torch_geometric/graphgym/model_builder.py

```diff
@@ -1,12 +1,47 @@
+import warnings
+
 import torch
 
 from torch_geometric.graphgym.config import cfg
 from torch_geometric.graphgym.models.gnn import GNN
 from torch_geometric.graphgym.register import network_dict, register_network
 
+try:
+    from pytorch_lightning import LightningModule
+except ImportError:
+    LightningModule = object
```

Comment:
I think would be better to fall to `nn.Module`

Reply:
Good point. Fixed in https://github.com/pyg-team/pytorch_geometric/pull/4516.